config/initializers/rescue_from.rb

```diff
@@ -0,0 +1,22 @@
+require 'openstax_rescue_from'
+
+OpenStax::RescueFrom.configure do |config|
+  config.raise_exceptions = Rails.application.config.consider_all_requests_local
+
+  config.app_name = 'Exercises'
+  config.app_env = ENV['APP_ENV']
+  config.contact_name = Rails.application.secrets['exceptions']['contact_name']
+
+  # config.notifier = ExceptionNotifier
+
+  # config.html_error_template_path = 'errors/any'
+  # config.html_error_template_layout_name = 'application'
+
+  # config.email_prefix = "[#{app_name}] (#{app_env}) "
+  config.sender_address = Rails.application.secrets['exceptions']['sender']
+  config.exception_recipients = Rails.application.secrets['exceptions']['recipients']
+end
```

Comment:
make this config look pretty much exactly like what I did in tutor-server and exchange.  note in particular that it isn't 'exceptions' in the deployment stuff but 'exception'.